assets/js/googlesitekit/components-gm3/Checkbox.test.js

```diff
@@ -111,14 +111,14 @@ describe( 'Checkbox', () => {
 		expect(
 			container
 				.querySelector( 'md-checkbox' )
-				.getAttribute( 'data-aria-label' )
+				.getAttribute( 'aria-label' )
 		).toBe( 'Complex Label With 5 Sub Children' );
 	} );
 
 	it( 'should attach the onKeyDown handler when present', () => {
 		const onKeyDown = jest.fn();
 
-		const { getByRole } = render(
+		const { getByLabelText } = render(
```

Comment:
Use `getByLabelText()` rather than `getByRole()` to reflect the fact we're no longer setting the role directly.